docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_remove-analysis-interpolation.txt

```diff
@@ -0,0 +1,8 @@
+* The deprecated module ```iris.analysis.interpolate``` has been removed, along with the following functions:
+    * ```iris.analysis.linear```
+    * ```iris.analysis.all_points_in_range```
+    * ```iris.analysis.nearest_neighbour_data_value```
+    * ```iris.analysis.regrid```
+    * ```iris.analysis.extract_nearest_neighbour```
+    * ```iris.analysis.nearest_neighbour_indices```
+    * ```iris.analysis.Linear1dExtrapolator```
```

Comment:
You're missing `iris.analysis.interpolate.regrid_to_max_resolution` from this list.

Also, you need to add "`interpolate`" to each item in the list...